src/modules/UI/scenes/TransactionList/TransactionList.ui.js

```diff
@@ -265,77 +179,71 @@ export class TransactionList extends Component<Props, State> {
       fiatBalanceString = receivedFiatSymbol + ' ' + intl.formatNumber(balanceInFiat || 0, { toFixed: 2 }) + ' ' + fiatCurrencyCode
     }
     return (
-      <Animated.View style={[{ height: this.state.balanceBoxHeight }]}>
+      <TouchableOpacity onPress={this.props.toggleAccountBalanceVisibility} style={styles.touchablBalanceBox} activeOpacity={BALANCE_BOX_OPACITY}>
```

Comment:
why did you get rid of the animated view?

Reply:
Because the balance box used to need a fade (opacity change) for the search feature (that was never implemented), and since the area is now "pressable" that means we can use the `TouchableOpacity`'s native `activeOpacity` property to create the desired effect with improved performance.